Quotient/room.cpp

```diff
@@ -127,6 +130,7 @@ class Q_DECL_HIDDEN Room::Private {
     QList<User*> usersTyping;
     QHash<QString, QSet<QString>> eventIdReadUsers;
     QList<User*> usersInvited;
+    QMap<QString, QSharedPointer<RoomMember>> invites;
```

Comment:
Do you really need the map sorted?
```suggestion
    QHash<QString, QSharedPointer<RoomMember>> invites;
```